setup.py

```diff
@@ -25,6 +25,7 @@
 import smdebug
 
 DOCLINES = (__doc__ or "").split("\n")
+CURRENT_VERSION = __version__
```

Comment:
are we using this anywhere?

Reply:
not sure... I was under impression that is used but if it is not - happy to remove it.